hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3043,18 +3043,18 @@
 
   <property>
     <name>ozone.default.bucket.layout</name>
-    <value>OBJECT_STORE</value>
+    <value>LEGACY</value>
```

Comment:
How about keeping it empty and not include `LEGACY` into the supported values.
` <value/>`

Reply:
Done